gangplank/cmd/gangplank/pod.go

```diff
@@ -77,6 +87,21 @@ func runPod(c *cobra.Command, args []string) {
 	cluster := ocp.NewCluster(true)
 
 	if cosaViaPodman {
+		if cosaViaRemotePodman != "" {
```

Comment:
I think we need something slightly different here. We basically want to support someone implying `--remote` by setting `CONTAINER_HOST` ahead of time. Something like this should help I think:

```patch
diff --git a/gangplank/cmd/gangplank/pod.go b/gangplank/cmd/gangplank/pod.go
index cc058839..c74bf1bd 100644
--- a/gangplank/cmd/gangplank/pod.go
+++ b/gangplank/cmd/gangplank/pod.go
@@ -87,8 +87,16 @@ func runPod(c *cobra.Command, args []string) {
        cluster := ocp.NewCluster(true)
 
        if cosaViaPodman {
+               // Whether provided via CLI or env var let's sync them up now
+               // so that the env var and local variable have the same contents.
+               // If both are defined the CLI (--remote) takes precedence.
                if cosaViaRemotePodman != "" {
                        os.Setenv(podmanRemoteEnvVar, cosaViaRemotePodman)
+               } else if x := os.Getenv(podmanRemoteEnvVar); x != "" {
+                       cosaViaRemotePodman = x
+               }
+
+               if cosaViaRemotePodman != "" {
                        minioSshRemoteHost = containerHost()
                        if strings.Contains(minioSshRemoteHost, "@") {
                                parts := strings.Split(minioSshRemoteHost, "@")
```

Reply:
Alternatively we just make the var default to `os.Getenv(podmanRemoteEnvVar)` rather than `""`

Reply:
I'm going to go with a default.

Reply:
Okay, repushed.